File: packages/web-app/src/models/Project.ts
export interface Project {
    id: number;
    name: string;
    description: string;
    // url: string;
    // avatarUrl?: string;
}

export function isProject(result: Project | string): result is Project {
    return !!(result as Project).id;
}

export function isProjectArray(result: Project[] | string): result is Project[] {
    return Array.isArray(result);
}

File: packages/lts/src/main/java/com/bt/scrumbleLts/dto/Project.java
package com.bt.scrumbleLts.dto;

public class Project {
    private final int id;
    private final String name;
    private final String description;
    private final String avatar_url = null;

    public Project(int id, String name, String description) {
        this.id = id;
        this.name = name;
        this.description = description;
    }
}
